### DISCUSSION TITLE
How to change path to temp files %temp%/FEDOT?

### DISCUSSION THREAD

**Discussion by geniusserg** [nccr-itmo/FEDOT#1002] [Q&A] [answered]
Hello!

During model optimization, Fedot framework created folder C:\Users\<User folder>\AppData\Local\Temp\FEDOT with databases with size more than 17Gb. How can I change path to these temp files? On C:/ part I haven't enough free space for storing them.

Thank you!

**Comment by nicl-nno** [comment, marked as answer]
Hi!

You can use change cache_path as decribed in https://fedot.readthedocs.io/en/latest/faq/api_usage.html
